lahja/asyncio/endpoint.py

```diff
@@ -462,7 +462,10 @@ def _stop_server(self) -> None:
         for task in self._server_tasks:
             task.cancel()
 
-        self.ipc_path.unlink()
+        try:
+            self.ipc_path.unlink()
+        except FileNotFoundError:
```

Comment:
Might still be worth to issue a warning or debug message? Just an idea, feel free to ignore.